build.sh

```diff
@@ -1,7 +1,19 @@
-git clone https:\/\/github.com\/fossasia\/fossasia.org.git out
+git config --global push.default simple
+ssh-agent bash -c "ssh-add ${travis_deploy_key}"
```

Comment:
Also, you can do the remote add on the current repository you are in. Do not name it origin but differently.